DigitalLearningSolutions.Web/Views/Shared/_Layout.cshtml

```diff
@@ -6,14 +6,14 @@
 @inject IConfiguration Configuration
 
 @{
-  var headerExtension = (string)ViewData[LayoutViewDataKeys.Application];
-  var shouldDisplayHelpMenuItem = headerExtension != "Tracking System";
   var dlsSubApplication = (DlsSubApplication)ViewData[LayoutViewDataKeys.DlsSubApplication];

```

Comment:
Just noticed, I think you should be casting this to a `DlsSubApplication?` because it might be null. Does that work okay?

Reply:
I think it should work fine, just not certain because we've written the implicit cast ourself.

Reply:
Oh right with the ? at the end, let me check